controls/combobox/editmode-and-search.md

```diff
@@ -1,33 +1,33 @@
-description: Check our &quot;Edit Mode &amp; Search&quot; documentation article for Telerik UI for .NET MAUI ComboBox.
+description: Learn more about the edit mode and search functionalities in Telerik UI for .NET MAUI ComboBox control.
 position: 4
 slug: combobox-editmode-and-search
 ---
 
 # .NET MAUI ComboBox Edit Mode & Search
 
-ComboBox supports both **editable** and **noneditable** state. When the control is in edit mode, users can type into the input field and searching through the control's ItemsSource can be performed. 
+ComboBox supports both `editable` and `non editable` state. When the control is in edit mode, users can type into the input field and searching through the control's ItemsSource can be performed. 
 
 This topic describes in details the configuration options related to the editable state of the ComboBox as well as its searching capabilities.
 
 ## Edit Mode
 
 * `IsEditable`(`bool`)&mdash;Defines whether typing into the input area can be performed. The default value is `false`.
 * `SearchTextPath`(`string`)&mdash;Defines the name of the property against which the searching will be performed. The property is usable when the control is in edit mode (`IsEditable` is `true`). 
-* `OpenOnFocus`(`bool`)&mdash;Specifies whether the dropdown will be shown as soon as the control is focused. The property is usable when the control is in edit mode (`IsEditable` is `true`).  The default value is `true`. If the property is set to false when the control is focused the drop-down will no longer open.
-* `Text`(`string`)&mdash;Specifies the Text of the control. This is the Text that gets visualized when the control is editable or when it is non-editable and the selection mode is single.
+* `OpenOnFocus`(`bool`)&mdash;Specifies whether the dropdown will show as soon as the control is focused. The property is usable when the control is in edit mode (`IsEditable` is `true`).  The default value is `true`. If the property is `false` when the control is focused the drop-down will no longer open.
+* `Text`(`string`)&mdash;Specifies the Text of the control. This is the Text that gets visualized when the control is editable or when its non-editable and the selection mode is single.
 
 ## Search
 
 ComboBox provides both case-sensitive and case-insensitive searching modes. The following properties are exposed:
 
-- `SearchMode`(enumeration of type *Telerik.Maui.Controls.SearchMode*)&mdash;Defines the value that sets search sets some search criteria for the control. The available options are: `Contains`, `StartsWith`, `ContainsCaseSensitive` and `StartsWithCaseSensitive`. The default SearchMode is `StartsWith`.
+- `SearchMode`(enumeration of type `Telerik.Maui.Controls.SearchMode`)&mdash;Defines the value that sets search sets some search criteria for the control. The available options are: `Contains`, `StartsWith`, `ContainsCaseSensitive` and `StartsWithCaseSensitive`. The default SearchMode is `StartsWith`.
```

Comment:
The first sentence is incorrect, it doesn't make sense. "Defines the value that sets search sets some search criteria for the control." Something is repeating or missing.